app/controllers/account_reset/confirm_request_controller.rb

```diff
@@ -0,0 +1,12 @@
+module AccountReset
+  class ConfirmRequestController < ApplicationController
+    def show
+      if session[:email].blank?
+        redirect_to root_url
+      else
+        email = session.delete(:email)
```

Comment:
This looks like it's behaving similar to the `flash`.  Would the following behave the same?
```ruby
def show
  if flash[:email].blank?
    redirect_to root_url
  else
    render :show, locals: { email: flash[:email] }
  end
end
```

That keeps the view content exactly the same, but I *think* it will pass the value from one request to the next and remove it on the next request. If we're bouncing through several pages before getting here, then that might not work.

Reply:
I assumed that flash would not be safe to store email, etc.  From a little reading it seems to be stored safely in the session.  Is this correct?